package.json

```diff
@@ -13,6 +13,10 @@
       "default": "reek"
     }
   },
+  "activationHooks": [
+    "language-ruby:grammar-used",
```

Comment:
The point of https://github.com/AtomLinter/Meta/issues/20 is to use the `scope.name:root-scope-used` hook, _specifically_ to avoid the need to list out the name of all the different grammar packages that provide that scope.